eng/targets/Settings.props

```diff
@@ -12,7 +12,9 @@
 
     <CommonExtensionInstallationRoot>CommonExtensions</CommonExtensionInstallationRoot>
     <LanguageServicesExtensionInstallationFolder>Microsoft\ManagedLanguages\VBCSharp\LanguageServices</LanguageServicesExtensionInstallationFolder>
-    
+
```

Comment:
Can we avoid blank lines before after properties in property blocks? Unless there is a comment prefer we just group them together.